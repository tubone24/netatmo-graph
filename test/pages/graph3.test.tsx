import React from 'react'
import { render } from '../testUtils'
import Graph3 from '../../pages/graph3'
import { RecoilRoot } from 'recoil'

describe('Graph3', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <RecoilRoot>
        <Graph3 />
      </RecoilRoot>,
      {}
    )
    // https://github.com/mui-org/material-ui/issues/21293#issuecomment-654921524
    expect(
      asFragment()
        .firstElementChild.innerHTML.replace(/id="mui-[0-9]*"/g, '')
        .replace(/aria-labelledby="(mui-[0-9]* *)*"/g, '')
    ).toMatchSnapshot()
  })
})
