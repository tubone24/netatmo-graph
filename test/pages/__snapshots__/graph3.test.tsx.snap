// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Graph3 matches snapshot 1`] = `"<main class=\\"jsx-531684522\\"><h1 class=\\"jsx-531684522 title\\">Back to <a href=\\"/\\">AWS Health Dashboard</a></h1><div class=\\"container\\"><div class=\\"container\\"><canvas height=\\"0\\" width=\\"0\\" style=\\"display: block;\\" class=\\"chartjs-render-monitor\\"></canvas></div></div></main>"`;
